Makefile

```diff
@@ -87,16 +109,28 @@ run:
 	  --env INFERENCE_MODEL=$(ANSIBLE_CHATBOT_INFERENCE_MODEL) \
 	  ansible-chatbot-stack:$(ANSIBLE_CHATBOT_STACK_VERSION)
 
-run-local-db:
+# Pre-check required environment variables for local DB run
+check-env-run-local-db: check-env-run
+	@if [ -z "$(LOCAL_DB_PATH)" ]; then \
+		echo "$(RED)Error: LOCAL_DB_PATH is required but not set$(NC)"; \
+		exit 1; \
+	fi
+	@if [ -z "$(CONTAINER_DB_PATH)" ]; then \
+		echo "$(RED)Error: CONTAINER_DB_PATH is required but not set$(NC)"; \
+		exit 1; \
+	fi
+
+run-local-db: check-env-run-local-db
 	@echo "Running Ansible Chatbot Stack container..."
 	@echo "Using vLLM URL: $(ANSIBLE_CHATBOT_VLLM_URL)"
 	@echo "Using inference model: $(ANSIBLE_CHATBOT_INFERENCE_MODEL)"
+	@echo "Mapping local DB from $(LOCAL_DB_PATH) to $(CONTAINER_DB_PATH)"
 	docker run --security-opt label=disable -it -p $(LLAMA_STACK_PORT):$(LLAMA_STACK_PORT) \
+	  -v $(LOCAL_DB_PATH):$(CONTAINER_DB_PATH) \
 	  --env LLAMA_STACK_PORT=$(LLAMA_STACK_PORT) \
-	  --env VLLM_URL=$(ANSIBLE_CHATBOT_VLLM_URL) \
```

Comment:
You've removed this line (by accident?)

`make run-local-db` does not work without it:
```
--env VLLM_URL=$(ANSIBLE_CHATBOT_VLLM_URL) \
```